packages/kit/test/apps/basics/src/routes/redirect-on-load/_tests.js

```diff
@@ -0,0 +1,14 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test, is_dev) {
+	test('redirect-on-load', '/redirect-on-load', async ({ base, page, js }) => {
+		if (js) {
+			await page.waitForTimeout(50);
```

Comment:
I would not use this. Per the docs for that method:
>Note that page.waitForTimeout() should only be used for debugging. Tests using the timer in production are going to be flaky. Use signals such as network events, selectors becoming visible and others instead.

Reply:
Yes. Totally agree with you.
But already few exists tests use `waitForTimeout` so I followed that.

Then at least I will update this PR.
But should I refactor exists `waitForTimeout`?
I think we should do it to avoid random test failures.
If yes, I will create one more PR which is for existing tests with `waitForTimeout`.

Reply:
Yes, that would be great to fix the other tests so that they don't use `waitForTimeout`. Thank you! I filed an issue to track it: https://github.com/sveltejs/kit/issues/2473

Reply:
Oh, Thank you for creating a new issue. I will handle it.

Reply:
I have added a function to avoid the use of  `waitForTimeout`.
I hope to get your review.

And I will create a new PR for removing all `waitForTimeout` based on this branch.